kustomize/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM kubespheredev/builder-base:v3.1.0
+
+ENV KUSTOMIZE_VER 4.3.0
+
+RUN cd && \
```

Comment:
Please refer to this hack file: https://github.com/kubesphere/devops-agent/blob/master/base/hack/install_utils.sh.

IMO, there is no need a separated docker image for users to use `kustomize`.

Reply:
Do you mean, kustomize should be included within the base-image?

Reply:
Indeed, kustomize is already integrated into kubectl. Please close my issue.

Reply:
hi @xiabai84 ， I'm wondering if we can put `kustomize` into `builder-base`